libsolidity/ast/ASTForward.h

```diff
@@ -95,6 +95,8 @@ using ASTPointer = std::shared_ptr<T>;
 
 using ASTString = std::string;
 
+// How the function can mutate the EVM state.
+enum class StateMutability { Pure, View, NonPayable, Payable };
```

Comment:
Let's create an `ASTENums.h` and move this  `stateMutabilityToString()` or overload `to_string()`.